lib/authorizer/ntlm.js

```diff
@@ -150,7 +149,7 @@ module.exports = {
      * @param {AuthHandlerInterface~authPreHookCallback} done -
      */
     pre: function (auth, done) {
-        !auth.get(STATE) && auth.set(STATE, STATES.INITIALIZED);
+        !auth.getAuthState(NTLM) && auth.setAuthState(NTLM, STATES.INITIALIZED);
```

Comment:
Why changing the key?
```suggestion
        !auth.getAuthState(STATE) && auth.setAuthState(STATE, STATES.INITIALIZED);
```

Reply:
getAuthState / setAuthState We are using it only for the state, so it doesn't make sense to store it inside the `state` key 
the final populated object looks like this
```
{
     NTLM: 'INITIALIZED'
}
```

Reply:
But this is the state, no? Why not store in `state` itself? Naming it like `ntlm` feels weird if more state properties are added.